### ISSUE TITLE
About  3D ->2D  ？

### ISSUE THREAD

**Issue by zhanghongyong123456** [lisiyao21/AnimeRun#2]
Can it be reversed, not 3D to 2D, but 2D to 3D

**Comment by lisiyao21** [comment]
Yes. Thank you for mentioning that. We also made the depth and normal data of the 2D cartoon so that it can be extended to 2D->3D task. But we are still working on cleaning the data.

**Comment by zhanghongyong123456** [comment]
> Yes. Thank you for mentioning that. We also made the depth and normal data of the 2D cartoon so that it can be extended to 2D->3D task. But we are still working on cleaning the data.

Thank you very much for your reply. When will it be released? I am looking forward to it
